geotrek/feedback/locale/fr/LC_MESSAGES/django.po

```diff
@@ -219,8 +225,8 @@ msgstr "De nouveaux signalements ont été importés depuis Suricate."
 msgid "Please consult your reports in Geotrek-Admin."
 msgstr "Consultez vos signalements dans Geotrek-Admin."
 
-msgid "Geotrek - Un Signalement est à clôturer"
-msgstr "Geotrek - A Report is ready to be resolved"
+msgid "Un Signalement est à clôturer"
+msgstr "A Report is ready to be resolved"
```

Comment:
translations are inverted ? msgid in English then msgstr in French ?

Reply:
Oh. This entry isn't used anyway, I'm deleting it